domain-models-runtime/src/main/java/org/folio/rest/persist/Conn.java

```diff
@@ -866,4 +867,21 @@ static String durationMsg(String descriptionKey, String sql, long startNanoTime)
 
     return streamGet(table, clazz, PostgresClient.DEFAULT_JSONB_FIELD_NAME, wrapper, false, null, null, streamHandler);
   }
+
+  /**
+   * Get a stream of the results of the {@code sql} query.
+   *
+   * @param params arguments for {@code $} placeholders in {@code sql}
+   * @param chunkSize cursor fetch size
+   */
+  public Future<RowStream<Row>> selectStream(String sql, Tuple params, int chunkSize) {
```

Comment:
Will there be a futurized version of `selectStream` in `PostgresClient`?

Reply:
I've added it.